java-logging-appinsights/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Application Insights
 
-Logging module to auto configure Request Name Interceptor and Web Request Tracking Filter for Azure Application Insights app. Module also tracks logback INFO events for `uk.gov.hmcts` loggers
+Logging module to auto configure necessary components for Azure Application Insights app. It uses [azure-application-insights-spring-boot-starter](https://github.com/Microsoft/ApplicationInsights-Java/tree/master/azure-application-insights-spring-boot-starter) and adds some specific initializers as Spring beans. Module also tracks logback INFO events for `uk.gov.hmcts` loggers
```

Comment:
We can point to actual [version (check url)](https://github.com/Microsoft/ApplicationInsights-Java/tree/v2.3.1/azure-application-insights-spring-boot-starter) we are using and it'll be automatically updated by version bumper script 🙃. oh wait a minute, it's for library itself. anyway, small suggestion to consider about actual link

Reply:
Done, not sure if i am introducing a maintenance night mare by this.